plaso/formatters/chrome.py

```diff
@@ -89,9 +89,9 @@ def GetMessages(self, unused_formatter_mediator, event):
     """Determines the formatted message strings for an event object.
 
     Args:
-      formatter_mediator (FormatterMediator): mediates the interactions between
-          formatters and other components, such as storage and Windows EventLog
-          resources.
+      unused_formatter_mediator (FormatterMediator): mediates the interactions
```

Comment:
Let's discuss IMHO the argument in the docstring should be named `formatter_mediator` not `unused_formatter_mediator`. the unused prefix is an artifact of pylint